### ISSUE TITLE
Accessibility Issue : Actionable images have no alternate text

### ISSUE THREAD

**Issue by Developer-BHN** [uploadcare/file-uploader#773]
**Description**
Some actionable images do not have alternate text.

Actionable images must have alternate text. Assistive technology users will not be able to determine the image's purpose, and therefore the link or button's purpose.

**Issue Code**
```html
<button type="button" class="uc-mini-btn uc-close-btn">
  <uc-icon name="close">
    <svg xmlns="http://www.w3.org/2000/svg">
      <title></title>
      <use href="#uc-icon-close"></use>
    </svg>
  </uc-icon>
</button>
```

**Recommended Code**
```html
<button type="button" aria-label="close" class="uc-mini-btn uc-close-btn">
  <uc-icon name="close" aria-hidden="true">
    <svg xmlns="http://www.w3.org/2000/svg">
      <title></title>
      <use href="#uc-icon-close"></use>
    </svg>
  </uc-icon>
</button>
```

**Remediation Guidance**

Ensure all actionable images have alternate text.

**Screenshots**
<img width="409" alt="DE33208_1" src="https://github.com/user-attachments/assets/ee6e6318-ba55-4c74-9862-f7cd06f49a71" />
<img width="500" alt="DE33208_2" src="https://github.com/user-attachments/assets/e9879dd9-de4b-4e98-9333-7ab8998cefcf" />

**Issue Metadata**

Applicable Standards:

1.3.1 Info and Relationships (Level A)
2.4.4 Link Purpose (In Context) (Level A)
4.1.2 Name, Role, Value (Level A)
 

**Content Type:**

error

Impacted Populations

Blind (Severity: 3)Low Vision (Severity: 2)Motor Impaired (Severity: 3)Cognitively Impaired (Severity: 2)

Issue Key:

94
